apps/dashboard/src/app/account/account-manage/account-delete/account-delete.component.scss

```diff
@@ -10,12 +10,12 @@ label {
 }
 
 .delete-user {
-  margin: rhythm(4) 0;
+  margin: rhythm(4) 0 rhythm(3) 0;
 }
 
-pa-radio-group > div {
```

Comment:
oh I missed this on my first review! you don't need to change your new style for this, but just for you to know, you could have done what you wanted without the `div`, by adding an `!important` on the display property, like:

```scss
pa-radio-group {
  display: flex !important;
  gap: rhythm(5);
}
```

Reply:
I should try to find another way to set the default display block on radio group so we don't need this `!important` though